packages/amplify-e2e-tests/src/__tests__/import-rds-v2-1.test.ts

```diff
@@ -0,0 +1,496 @@
+    const { dbAdapter, dbInfo } = await setupRDSDatabase({
+      username: db_user,
+      password: db_password,
+      database: db_name,
+      identifier: identifier,
+      region: region
+    });
+    dbConnectionInfo.port = dbInfo.port;
+    dbConnectionInfo.publicIpCidr = dbInfo.publicIpCidr;
+    dbConnectionInfo.host = dbInfo.host;
+    await dbAdapter.runQuery(sqlStatements);
+  });
+
+  afterAll(async () => {
+    dbAdapter?.cleanup();
+    await removeRDSPortInboundRule({
+      region,
+      port: dbConnectionInfo.port,
+      cidrIp: dbConnectionInfo.publicIpCidr
+    });
+    await deleteDBInstance(identifier, region);
+  });
+
+  beforeEach(async () => {
+    projectRoot = await createNewProjectDir("import-rds-env-v2");
+  });
+
+  afterEach(async () => {
+    await deleteProject(projectRoot);
+    deleteProjectDir(projectRoot);
+  });
+
+  it("adds a new api if one does not already exist", async () => {
+    const name = 'importnewapi';
+    await initJSProjectWithProfile(projectRoot, { disableAmplifyAppCreation: false, name: name });
+    await importRDSDatabase(projectRoot, {
+      database: db_name,
+      host: dbConnectionInfo.host,
+      port: dbConnectionInfo.port,
+      username: db_user,
+      password: db_password,
+      apiExists: false
+    });
+    await amplifyPush(projectRoot);
+
+    verifyAmplifyMeta(projectRoot, name, db_name);
+    verifyCompiledSchema(projectRoot, name, expectedCompiledSchema);
+    verifyRDSSchema(projectRoot, name, expectedSchema);
+  });
+
+  it("uses the existing api if one exists", async () => {
+    const name = 'importexistingapi';
+    await initJSProjectWithProfile(projectRoot, { disableAmplifyAppCreation: false, name: name });
+    await addApiWithBlankSchema(projectRoot);
+    await updateApiSchema(projectRoot, name, 'simple_model.graphql');
+    await importRDSDatabase(projectRoot, {
+      database: db_name,
+      host: dbConnectionInfo.host,
+      port: dbConnectionInfo.port,
+      username: db_user,
+      password: db_password,
+      apiExists: true
+    });
+    await amplifyPush(projectRoot);
+
+    verifyAmplifyMeta(projectRoot, name, db_name);
+    verifyCompiledSchema(projectRoot, name);
+    verifyRDSSchema(projectRoot, name, expectedSchema);
+  });
+
+  it("removing the API clears out the imported schema file", async () => {
+    const name = "wrongsecrets";
+    await initJSProjectWithProfile(projectRoot, { disableAmplifyAppCreation: false, name: name });
+    const dbInfo = {
+      database: db_name,
+      host: dbConnectionInfo.host,
+      port: dbConnectionInfo.port,
+      username: db_user,
+      password: db_password
+    };
+    await importRDSDatabase(projectRoot, {
+      ...dbInfo,
+      apiExists: false
+    });
+
+    verifyRDSSchema(projectRoot, name, expectedSchema);
+    
+    // remove the API
+    await removeApi(projectRoot);
+    verifyRDSSchema(projectRoot, name);
+  });
+});
+
+const expectedSchema: string = `
```

Comment:
It seems that the schema is the same as the one in `v2-2`. If so, this can be defined in the one shared place.

Reply:
Yes we're using the same schema, but would probably refine the schemas as we add more tests in each file. I couldn't find a better place/pattern to share these across tests which is probably because it helps readability if each test holds it's own artifacts.